src/app/App.module.css

```
@@ -0,0 +1,39 @@
+.container {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  place-items: center;
+}
+
+.image {
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+}
+
+.form {
+  display: grid;
+  row-gap: 1rem;
+}
+
+.input {
+  font-size: 4rem;
+  text-align: center;
+}
+
+.submitbutton {
```

```suggestion
.submitButton {
```
Don't forget to use camelCase or some other naming convention, but pick one and stay consistent! 🐞